webserver/reply.cpp

```diff
@@ -280,9 +244,11 @@ void reply::add_header(reply *rep, const std::string &name, const std::string &v
 {
 	size_t num = rep->headers.size();
 	if (replace) {
-		for (size_t h = 0; h < num; h++) {
-			if (boost::iequals(rep->headers[h].name, name)) {
-				rep->headers[h].value = value;
+		for (auto &h : rep->headers)
+		{
+			if (h.name == name)
```

Comment:
boost::iequals is a case insensitive compare function. You are making it case sensitive which should not be the case.,
I already made a new function in the helper class and will make a quick patch before merging this PR